etc/init/spinnaker-monitoring.conf

```diff
@@ -0,0 +1,3 @@
+start on filesystem or runlevel [2345]
+
```

Comment:
How do you have your dd agent config protected?
I was running as root because spinnaker cannot read my dd agent config to get the API key.